.github/workflows/tests.yaml

```diff
@@ -34,9 +36,16 @@ jobs:
         uses: ramsey/composer-install@v1
         with:
           composer-options: --prefer-dist
+        if: matrix.php-version > 7.1
```

Comment:
@lstrojny not sure if it's a good idea or not. But since BC Checker version 1 also doesn't support PHP 7.1, it was failing on the Install Composer dependencies step for that version.
For now, I skipped all the steps for 7.1 assuming we will soon get rid of these older versions.
But is this the right way of doing it?